### PR TITLE
Add homepage featured section - red weather alert

### PR DESCRIPTION
**Pull request by richardTowers**
This temporarily replaces the War in Ukraine featured section.

| Before | After |
|----|----|
| Screenshot 2022-07-17 at 13 35 19 | image |

- [x] Get review from content on call - [reviewed on Zendesk](https://govuk.zendesk.com/agent/tickets/5025893)
- [ ] Get review from engineering on call

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -402,10 +402,10 @@ en:
           title: Get benefits and support
           href: /check-benefits-financial-support
           image_src: homepage/cost-of-living.jpg
-        - text: Find out about the UK's response to the Russian government's invasion of Ukraine and how you can help.
-          title: War in Ukraine
-          href: https://ukstandswithukraine.campaign.gov.uk/
-          image_src: homepage/ukraine-promo.jpg
+        - text: High temperatures can be harmful to your health. Learn how to stay safe in hot weather.
+          title: Staying safe in hot weather
+          href: https://www.gov.uk/government/publications/heatwave-plan-for-england/beat-the-heat-staying-safe-in-hot-weather
```
Comment (kevindew) [inline config/locales/en.yml:407]: Minor - convention is to have just a path for internal links (keeps it environment agnostic)

File: config/locales/en.yml [hunk 2]
```diff
@@ -402,10 +402,10 @@ en:
           title: Get benefits and support
           href: /check-benefits-financial-support
           image_src: homepage/cost-of-living.jpg
-        - text: Find out about the UK's response to the Russian government's invasion of Ukraine and how you can help.
-          title: War in Ukraine
-          href: https://ukstandswithukraine.campaign.gov.uk/
-          image_src: homepage/ukraine-promo.jpg
```
Comment (kevindew) [inline config/locales/en.yml:408]: We [have a convention](https://github.com/alphagov/frontend/blob/main/docs/update-homepage-promotion-slots.md#4-delete-the-previous-images) to delete this file

**Review by kevindew — APPROVED** [review]
Changes look good and will do the job. Some minor convention comments that can be resolved with an extra push or in working hours.

Also have you compressed the image? It seems pretty small at 40kb but we often aggressively compress to about 7-8kb but that might need a jpg. Also if you do tweak the image it does look like it’s maybe 1 pixel to wide (611 rather than 610) but that could just be what I’m using on my phone getting it wrong.

File: config/locales/en.yml (hunk 2 above)
Comment (richardTowers) [reply config/locales/en.yml:408]: Thanks, addressed in 1cac5b1

**Review by kevindew — APPROVED** [review]
Thanks

File: config/locales/en.yml (hunk 1 above)
Comment (richardTowers) [reply config/locales/en.yml]: Thanks, addressed in 41d5a70

**Pull request merged**